I'm sorry, but an uncaught exception occurred.

While loading <renpy.display.im.Image object ('gui/cursur.png') at 0x000000000541efd0>:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
OSError: Couldn't find file 'gui/cursur.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 211, in script call
    call _gl_test
  File "renpy/common/00gltest.rpy", line 388, in script
    $ __gl_test()
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00gltest.rpy", line 388, in <module>
    $ __gl_test()
  File "renpy/common/00gltest.rpy", line 317, in _m1_00gltest__gl_test
    _gl_performance_test()
  File "renpy/common/00gltest.rpy", line 340, in _gl_performance_test
    ui.interact(suppress_underlay=True, suppress_overlay=True)
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\core.py", line 2122, in interact
    self.start()
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\core.py", line 1036, in start
    surf = renpy.display.im.load_surface(fn)
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\im.py", line 2055, in load_surface
    return cache.get(image(im))
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\im.py", line 309, in get
    surf = image.load()
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\im.py", line 761, in load
    raise e
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\display\im.py", line 711, in load
    filelike = renpy.loader.load(self.filename, directory="images")
  File "C:\coding\renpy-8.2.0-sdk.7z\renpy-8.2.0-sdk\renpy\loader.py", line 672, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'gui/cursur.png'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.2.0.24012702
 1.0
Fri Mar  1 15:18:25 2024
